default/main.py

```diff
@@ -82,7 +83,8 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         logging.error(f'VT query failed with {vt_resp.status}: {vt_resp.text()}')
                         raise HTTPException(400, 'Bad request')
 
-                    async with httpSession.post('https://webhook-dot-virustotal-step-2020.ew.r.appspot.com/',
+                    # auth to webhook, get JWT token
```

Comment:
Maybe something like that?
```suggestion
                    # Get JWT token from webhook app.
```

Reply:
I think it's different, because I'm not requesting static token, I'm asking for a personal token after authorisation